Nos/Views/NoteCard.swift

```diff
@@ -181,9 +150,7 @@ struct NoteCard_Previews: PreviewProvider {
             }
         }
         .environment(\.managedObjectContext, previewData.previewContext)
-        .environmentObject(previewData.relayService)
         .environmentObject(previewData.router)
-        .environment(previewData.currentUser)
```

Comment:
This causes the preview to crash for me. Can you explain why you removed it?

Reply:
I removed it while removing a currentUser dependency that I added while working on it, I'll bring it back